backend/src/test/java/com/github/devtoju/backend/gaptext/GapTextFactory.java

```diff
@@ -11,68 +11,131 @@ public class GapTextFactory {
 
     static String description = "to clean, to ring";
     static String emptyDescription = "";
+    static String updatedDescription = "updated description";
     static String id = "testId";
 
     /**
-     * Creates a gap text container with four text items
+     * Instantiate container with four text items
      *
-     * @return A new instance of a gap text container
+     * @return A new gap text container
      */
-    public static GapTextContainer createContainer() {
+    public static GapTextContainer ofGapTextContainer() {
         return new GapTextContainer(id, description, createTexts());
     }
 
     /**
-     * Creates a gap text container DTO with four text items
+     * Instantiate container with four text items, a given id and an updated description
      *
-     * @return A new instance of a gap text container DTO
+     * @return A new gap text container
      */
-    public static GapTextContainerDTO createContainerDTO() {
-        return new GapTextContainerDTO(description, createTexts());
+    public static GapTextContainer ofGapTextContainerUpdated(String id) {
+        return new GapTextContainer(id, updatedDescription, createTexts());
     }
 
     /**
-     * Creates a gap text container DTO with four text items and
-     * an empty description
+     * Instantiate a container with four text items
      *
-     * @return A new instance of a gap text container DTO
+     * @return A new gap text container create DTO
      */
-    public static GapTextContainerDTO createContainerDtoWithEmptyDescription() {
-        return new GapTextContainerDTO(emptyDescription, createTexts());
+    public static GapTextContainerCreateDTO ofCreateDTO() {
+        return new GapTextContainerCreateDTO(description, createTexts());
     }
 
     /**
-     * Creates a gap text container DTO with an empty description and
+     * Instantiate a container with four text items and an empty description
+     *
+     * @return A new gap text container create DTO
+     */
+    public static GapTextContainerCreateDTO ofCreateDtoWithEmptyDescription() {
+        return new GapTextContainerCreateDTO(emptyDescription, createTexts());
+    }
+
+    /**
+     * Instantiate a container with an empty description and
      * an empty list of gap texts
      *
-     * @return A new instance of a gap text container DTO
+     * @return A new gap text container create DTO
      */
-    public static GapTextContainerDTO createContainerDtoWhereDescriptionAndGapTextsAreEmpty() {
-        return new GapTextContainerDTO(emptyDescription, Collections.emptyList());
+    public static GapTextContainerCreateDTO ofCreateDtoWhereDescriptionAndGapTextsAreEmpty() {
+        return new GapTextContainerCreateDTO(emptyDescription, Collections.emptyList());
```

Comment:
Weiß nicht ob das Absicht ist aber `Collections.emptyList()` ist immutable, da kann also nichts hinzugefügt werden

Reply:
Danke für den Hinweis mit dem immutable. Ist in diesem Fall nicht schlimm, da dieses Objekte einzig zum Testen erzeugt werden und zwar immer nur mit einer leeren Liste.